src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterOptions.cs

```diff
@@ -51,5 +51,30 @@ public sealed class ZipkinExporterOptions
         /// </summary>
         public int? MaxPayloadSizeInBytes { get; set; } = DefaultMaxPayloadSizeInBytes;
 #endif
+
+        /// <summary>
+        /// Gets or sets the exporter type for Zipkin Exporter.
+        /// </summary>
+        public ZipkinExporterType ExporterType { get; set; } = ZipkinExporterType.SimpleExportProcessor;
+
+        /// <summary>
+        /// Gets or sets the maxQueueSize for BatchExport.
+        /// </summary>
+        public int MaxQueueSize { get; set; } = 2048;
```

Comment:
Do we want to get the default value from the BatchExportProcessor?

Reply:
It makes it simpler to call the builder.AddProcessor(new BatchExportProcessor<Activity>(...) in ZipkinExporterHelperExtensions,cs. Otherwise we would have to keep a track of which options have been set by the user and find the right combination to call BatchExportProcessor. We can get default values from BatchExportProcessor but there will a lot of checks in the extension method then.

Reply:
what if BatchExportProcessor exposes public readonly const for its defaults? Like `public readonly int ExporterTimeoutMillisecondsDefault = 30000;`? Then we don't have to teach every exporter what is the default values, they can get it from BatchExportProcessor

Reply:
Added public constants to the BatchExportProcessor class.